hazelcast-jet-core/src/test/java/com/hazelcast/jet/MulticastDiscoveryTest.java

```diff
@@ -122,7 +130,13 @@ public void when_hazelcastClientCreated_then_doesNotConnectToJetCluster() {
 
         expectedException.expect(IllegalStateException.class);
         expectedException.expectMessage(UNABLE_TO_CONNECT_MESSAGE);
-        HazelcastClient.newHazelcastClient();
+
+        ClientConfig clientConfig = ClientConfig.load();
```

Comment:
looks like we duplicate retry logic, move it to a method perhaps

Reply:
Not sure, what you mean exactly. Those are 2 different configs & clients. I would say resemblance is accidental.

Reply:
I meant this part
```
private void configureTimeout(ClientConfig clientConfig) {
        clientConfig.getConnectionStrategyConfig()
                .getConnectionRetryConfig()
                .setClusterConnectTimeoutMillis(CLUSTER_CONNECTION_TIMEOUT);
}
```